_data/banking.yml

```diff
@@ -305,6 +305,13 @@ websites:
       tfa: Yes
       sms: Yes
 
+    - name: First Bank (Colorado)
```

Comment:
Their website uses `FirstBank` without a space. Could you please amend your pull request to reflect that? Also, I am not sure if `(Colorado)` is necessary.